docs: Update 005-liquidation.md (#1713)
umee-network/umee: docs/design_docs/005-liquidation.md

- Liquidation Incentive (10%)
- Close Factor (50%)

>   When a borrower is even 0.0001% over their borrow limit, they stand to lose value equal to 50% of their borrowed value in a single liquidation event.

Reviewer: ```suggestion
  When a borrower is even `0.0001%` over their borrow limit, a liquidator can repay `50%` of their borrowed USD value in a single liquidation event, receiving collateral worth `1 + (50% * 10%)` =`55%` of their borrowed value. This represents a loss of up to `5%` of the borrower's supplied value at the moment of liquidation. 
```

>   That is, the liquidator liquidates 50% of the borrowed value and receives 5% extra in collateral.

Reviewer: ```suggestion
```

- Liquidation Incentive (10%)
- Close Factor (50%)

>   When a borrower is even `0.0001%` over their borrow limit, a liquidator can repay `50%` of their borrowed USD value in a single liquidation event, receiving collateral worth `1 + (50% * 10%)` =`55%` of their borrowed value. This represents a loss of up to `5%` of the borrower's supplied value at the moment of liquidation. 

Reviewer: ```suggestion
  When a borrower is even `0.0001%` over their borrow limit, a liquidator can repay `50%` of their borrowed USD value in a single liquidation event, receiving collateral worth `1 + (50% * 10%)` = `55%` of their borrowed value. This represents a loss of up to `5%` of the borrower's supplied value at the moment of liquidation. 
```

>   When a borrower is even `0.0001%` over their borrow limit, a liquidator can repay `50%` of their borrowed USD value in a single liquidation event, receiving collateral worth `1 + (50% * 10%)` = `55%` of their borrowed value. This represents a loss of up to `5%` of the borrower's supplied value at the moment of liquidation. 

Reviewer: ```suggestion
  When a borrower is even `0.0001%` over their borrow limit, a liquidator can repay `50%` of their borrowed USD value in a single liquidation event, receiving collateral worth `1 + (50% * 10%)` = `55%` of their borrowed value.
 This represents a loss of up to `5%` of the borrower's supplied value at the moment of liquidation. 
```

- Liquidation Incentive (10%)
- Close Factor (50%)
  When a borrower is even `0.0001%` over their borrow limit, a liquidator can repay `50%` of their borrowed USD value in a single liquidation event, receiving collateral worth `1 + (50% * 10%)` = `55%` of their borrowed value.

>  This represents a loss of up to `5%` of the borrower's supplied value at the moment of liquidation. 

Reviewer 2: ```suggestion
 This represents a loss of up to `5%` of the borrower's supplied value at the moment of liquidation.
```
